<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>585</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSetupLOD">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Setup LOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCleanUp">
       <property name="text">
        <string>Clean Up</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <item>
        <widget class="QPushButton" name="btnSpreadHonrizonal">
         <property name="text">
          <string>Spread Honrizontal</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spnValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSpreadVertical">
         <property name="text">
          <string>Spread Vertical</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btnPreviousLOD">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;&lt;&lt; Previous LOD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNextLOD">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Next LOD &gt;&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD0">
         <property name="text">
          <string>LOD0</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD1">
         <property name="text">
          <string>LOD1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD2">
         <property name="text">
          <string>LOD2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD3">
         <property name="text">
          <string>LOD3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD4">
         <property name="text">
          <string>LOD4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD5">
         <property name="text">
          <string>LOD5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceLOD6">
         <property name="text">
          <string>LOD6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbSourceSHADOW">
         <property name="text">
          <string>SHADOW</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSwitchLOD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Switch LOD's</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD0">
         <property name="text">
          <string>LOD0</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD1">
         <property name="text">
          <string>LOD1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD2">
         <property name="text">
          <string>LOD2</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD3">
         <property name="text">
          <string>LOD3</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD4">
         <property name="text">
          <string>LOD4</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD5">
         <property name="text">
          <string>LOD5</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetLOD6">
         <property name="text">
          <string>LOD6</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbTargetSHADOW">
         <property name="text">
          <string>SHADOW</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnLOD0">
       <property name="text">
        <string>Type A</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD1">
       <property name="text">
        <string>Type B</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD2">
       <property name="text">
        <string>Type C</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD3">
       <property name="text">
        <string>Type D</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnLOD4">
       <property name="text">
        <string>Type Y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD5">
       <property name="text">
        <string>Type Z</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD6">
       <property name="text">
        <string>PULLED</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD7">
       <property name="text">
        <string>SMALL</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLOD8">
       <property name="text">
        <string>LARGE</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QCheckBox" name="chkBumper_front">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>bumper_front</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBumper_rear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>bumper_rear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkSide_skirt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>side_skirt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkHood">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>bonnet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
